classes/DirtSoundLibrary.sc

```diff
@@ -284,7 +337,15 @@ DirtSoundLibrary {
 			stretchInstrument: this.stretchInstrumentForBuffer(buffer),
 			bufNumFrames: buffer.numFrames,
 			bufNumChannels: buffer.numChannels,
-			unitDuration: { buffer.duration * baseFreq / ~freq.value },
+			baseFreqToMetaFreqRatio: baseFreqToMetaFreqRatio,
+			metaTuneRatio: {
```

Comment:
- [ ] maybe rename `metaTuneRatio` to `metaDataTuneRatio`? `metaTune` is ok, because it is short and we use it in tidal.